README.md

```diff
@@ -4,6 +4,16 @@
 
 _This product is in Alpha and not yet ready for production use. We welcome all feedback!_
 
+
+## Try it out!
+
+This script is not perfect but if you're on a linux-based system you should be able to get started by copying the following command into a terminal window and following the prompts:
+`
+curl -s https://raw.githubusercontent.com/DataBiosphere/job-manager/cjl_quick_start/deploy/quickstart/quick_start.py > /tmp/jmui.py && python3 /tmp/jmui.py
```

Comment:
I think it might be necessary to add a note here saying "you have to have python3 installed on your machine", otherwise  `python3 /tmp/jmui.py` will fail. Besides, `import urllib.request` will fail on Python2.x.